BinaryEncoding.md

```diff
@@ -786,9 +783,140 @@ for [future :unicorn:][future multiple tables] use and must be 0 in the MVP.
+| `i64.atomic_xor32_s` | `0xe61a` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic_xor32_u` | `0xe61b` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic_xchg` | `0xe610` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg` | `0xe611` | `5` `memory immediate` | atomic exchange with memory |
+| `i32.atomic_xchg8_s` | `0xe612` | `5` `memory immediate` | atomic exchange with memory |
+| `i32.atomic_xchg8_u` | `0xe613` | `5` `memory immediate` | atomic exchange with memory |
+| `i32.atomic_xchg16_s` | `0xe614` | `5` `memory immediate` | atomic exchange with memory |
+| `i32.atomic_xchg16_u` | `0xe615` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg8_s` | `0xe616` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg8_u` | `0xe617` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg16_s` | `0xe618` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg16_u` | `0xe619` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg32_s` | `0xe61a` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg32_u` | `0xe61b` | `5` `memory immediate` | atomic exchange with memory |
+
+## Atomic compare exchange operators ([described here](Semantics.md#atomic-memory-accesses))
+
+| Name | Opcode | Immediate | Description |
+| ---- | ---- | ---- | ---- |
+| `i32.atomic_cmpxchg` | `0xe61c` | `memory immediate` | atomic compare exchange with memory |
+| `i64.atomic_cmpxchg` | `0xe61d` | `memory immediate` | atomic compare exchange with memory |
+| `i32.atomic_cmpxchg8_s` | `0xe61e` | `memory immediate` | atomic compare exchange with memory |
+| `i32.atomic_cmpxchg8_u` | `0xe61f` | `memory immediate` | atomic compare exchange with memory |
+| `i32.atomic_cmpxchg16_s` | `0xe620` | `memory immediate` | atomic compare exchange with memory |
+| `i32.atomic_cmpxchg16_u` | `0xe621` | `memory immediate` | atomic compare exchange with memory |
+| `i64.atomic_cmpxchg8_s` | `0xe622` | `memory immediate` | atomic compare exchange with memory |
+| `i64.atomic_cmpxchg8_u` | `0xe623` | `memory immediate` | atomic compare exchange with memory |
+| `i64.atomic_cmpxchg16_s` | `0xe624` | `memory immediate` | atomic compare exchange with memory |
+| `i64.atomic_cmpxchg16_u` | `0xe625` | `memory immediate` | atomic compare exchange with memory |
+| `i64.atomic_cmpxchg32_s` | `0xe626` | `memory immediate` | atomic compare exchange with memory |
+| `i64.atomic_cmpxchg32_u` | `0xe627` | `memory immediate` | atomic compare exchange with memory |
+
+## Atomic load/store operators ([described here](Semantics.md#atomic-memory-accesses))
+
+| Name | Opcode | Immediate | Description |
+| ---- | ---- | ---- | ---- |
+| `i32.atomic_load` | `0xe628` | `memory immediate` | atomic load from memory |
+| `i64.atomic_load` | `0xe629` | `memory immediate` | atomic load from memory |
```

Comment:
I'm partial to calling the "atomic" prefix the "lock" prefix, and numbering it `0xF0`.